Dockerfile.prod

```diff
@@ -0,0 +1,33 @@
+# Build image
+FROM node:14-alpine as BUILDER
+
+WORKDIR /usr/src/app
+
+COPY package.json ./
+COPY yarn.lock ./
+
+RUN yarn install --frozen-lockfile --link-duplicates
+
+COPY . ./
+
+RUN yarn build
+
+
+
+# Production image
```

Comment:
This second part only copies the dist folder and the other necessary files for production;

- package.json
- yarn.lock
- ormconfig.js
- dist/ folder